examples/README.md

```diff
@@ -13,6 +13,7 @@ NODE_PATH=../ node examples/search.js
 ## Web scraping
 - [Puppetron](https://github.com/cheeaun/puppetron) - Demo site that shows how to use Puppeteer and Headless Chrome to render pages. Inspired by [GoogleChrome/rendertron](https://github.com/GoogleChrome/rendertron).
  - [Thal](https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e "An article on medium") - Getting started with Puppeteer and Chrome Headless for Web Scraping.
+ - [pupperender](https://github.com/LasaleFamine/pupperender) - ExpressJs middleware that checks the User-Agent header of incoming requests, and if it matches one of a configurable set of bots, render the page using Puppeteer. Useful for PWA rendering.
```

Comment:
ExpressJs -> Express

Reply:
bots -> search crawlers

maybe?

Reply:
Yeah, I was thinking to add another main area. I'll go for search crawlers, we can change it later.

Reply:
Yea but 'Puppetron' is also for rendering. Let's not add a new category and rename the existing title `## Rendering and web scraping`

Reply:
Make sense 👍

Reply:
PS: I fixed a little space left at the start of the Thal's article line.